scripts/setup-env.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable consistent-return, no-console */
+
+'use strict';
+
+const fs = require('fs');
+const { execSync } = require('child_process');
+const path = require('path');
+
+function executeCommand(command) {
+  console.log(`Executing command ${command}`);
+  execSync(command);
+}
+
+function copyConfig() {
```

Comment:
These files are already created by post install scripts of each sub package, and invoked by the root `npm install`.  So it shouldn't be necessary to copy them again?